app/views/users/search.html.erb

```diff
@@ -1,42 +1,51 @@
-<div class="jumbotron">
-    <h2><%= I18n.t('user.lists') %></h2>
+<%= stylesheet_link_tag 'users', media: 'all', 'data-turbolinks-track': 'reload' %>
 
-      <%# https://getbootstrap.com/docs/5.0/layout/grid/#auto-layout-columns %>
+<div class="stack">
+  <%= form_tag(search_users_path, method: "get", class: "form-tag justify-content-between") do %>
+    <%= text_field_tag :search, params[:search] , placeholder: I18n.t('user.search.place_holder'), class: "text-field form-control my-2 my-sm-0"%>
+    <%= submit_tag I18n.t('user.search.index') , class: "submit-button btn btn-outline-secondary my-2 my-sm-0"%>
+  <% end %>
 
-
-    <div class="row">
-        <div class="col">
-          <%= form_tag(search_users_path, method: "get") do %>
-            <%= label_tag(:search, I18n.t('user.search.for') ) %>
-            <%= text_field_tag :search, params[:search] %>
-            <%= submit_tag I18n.t('user.search.index') %>
+  <% @contacts.each do |user| %>
+    <div class=contact>
+      <label class="contact-info">
+        <%= link_to "", user_path(user) %>
+        <div class="profile-image my-2 my-sm-0 ">
+          <img src="//lh3.googleusercontent.com/-6V8xOA6M7BA/AAAAAAAAAAI/AAAAAAAAAAA/rzlHcD0KYwo/photo.jpg?sz=120" class="rounded-circle cover">
+        </div>
+        <div class="user-info my-2 my-sm-0">
+          <h5><%= "#{user.firstname} #{user.lastname}"%></h5>
+          <div class="email"><%= user.email %></div>
+        </div>
+      </label>
+        <div class="social-account-info">
+          <% user.social_accounts.each do |social_account| %>
+            <label class="contact-info">
+              <a href=<%= generate_link(social_account) %>>
+                <i class="fa <%= generate_icon(social_account) %> fa-2x" aria-hidden="true"></i>
+              </a>
+            </label>
           <% end %>
         </div>
     </div>
+  <% end %>
 
-    <table>
-        <thead>
-            <tr>
-                <th><%= I18n.t('user.attributes.username') %></th>
-                <th><%= I18n.t('user.attributes.firstname')%></th>
-                <th><%= I18n.t('user.attributes.lastname')%></th>
-                <th><%= I18n.t('user.attributes.email')%></th>
-            </tr>
-        </thead>
-        <tbody>
-
-        <% @users.each do |user| %>
-          <tr>
-            <td><%= user.username%></td>
-            <td><%= user.firstname%></td>
-            <td><%= user.lastname%></td>
-            <td><%= user.email %></td>
-            <% if @users_to_add.include? user %>
-              <td><%=  button_to '+', user_contact_requests_path(user), method: :post, id: user.id  %></td>
-            <% end %>
-          </tr>
+  <% @users.each do |user| %>
+    <div class=contact>
+      <label class="contact-info">
+        <div class="profile-image my-2 my-sm-0 ">
```

Comment:
This is almost the same code as for contacts above

Reply:
I know 🙈 . Just wanted to wait for the implementation of  #158 . Otherwise, I would have implemented some genric partials.

Reply:
#158 has been merged now